src/devices/tuya.ts

```diff
@@ -6373,6 +6373,76 @@ export const definitions: DefinitionWithExtend[] = [
                 [100, 160].includes(device.applicationVersion) || ["1.0.5\u0000"].includes(device.softwareBuildID), // polling for energy
             ),
     },
+    {
+        fingerprint: tuya.fingerprint("TS011F", ["_TZ321A_arrqgd67", "_TZ3210_2uollq9d", "_TZ3210_5ct6e7ye"]),
+        model: "TS011F_socket",
+        description: "Socket with power monitoring",
+        vendor: "Tuya",
+        whiteLabel: [
+            tuya.whitelabel("EKF", "RCS-ST16-WD-ZB", "Smart Socket Stockholm", ["_TZ321A_arrqgd67"]),
+            tuya.whitelabel("BSEED", "GL86ZEUSKM1W", "ZigBee EU Wall Socket", ["_TZ3210_2uollq9d"]),
+            tuya.whitelabel("BSEED", "GL86ZEUSKM1PDAC1W", "ZigBee EU Wall Socket Type-C With USB", ["_TZ3210_5ct6e7ye"]),
+        ],
+        ota: false,
+        extend: [
```

Comment:
```suggestion
        extend: [
           m.electricityMeter(),
```

Can you try with this and remove the `configure`?

Reply:
Sure, I'll give it a try on a dev instance later this month. Seems like a more concise way to configure reporting and other power meter settings.

Reply:
I'm wondering if we can integrate these changes into the already existing `TS011F_plug_1`, should prevent some code duplication.

Reply:
Do you see state-aware code being moved to fz converters? Specifically to reset power/current to 0 if state=OFF in `electrical_measurement` and `on_off` converters.

Reply:
That not, but for `TS011F_plug_1` it's fine.